tests/app.py

```diff
@@ -88,7 +89,7 @@ def header():
 # Thanks to bdarnell for the idea: https://github.com/pycurl/pycurl/issues/124
 @app.route('/header_utf8')
 def header_utf8():
-    header_value = bottle.request.headers[bottle.request.query['h']]
+    header_value = bottle.request.headers.get(bottle.request.query['h'], '' if py3 else b'')
```

Comment:
Out of curiosity are you testing pycurl on python 2?

Reply:
No, Python 3 only. Pycurl specifies Python 3 as a requirement, so I assumed Python 2 related code is just legacy. I only added that ternary expression because the rest of that function still handles Python 2.